pkg/server/endpoints/registration/handler.go

```diff
@@ -536,6 +536,46 @@ func (h *Handler) FetchBundle(
 	}, nil
 }
 
+//Evict removes (de-attest) node from attested nodes store
+func (h *Handler) EvictAgent(ctx context.Context, evictRequest *registration.EvictAgentRequest) (*registration.EvictAgentResponse, error) {
+	spiffeID := evictRequest.GetSpiffeID()
+	err := h.deleteAttestationEntry(ctx, spiffeID)
+	if err != nil {
+		h.Log.Warnf("Fail to de-attested agent with ID: %q", spiffeID)
+		return &registration.EvictAgentResponse{
+			DeleteSucceed: false,
+		}, err
+	}
+
+	h.Log.Infof("Successfully de-attested agent with ID: %q", spiffeID)
```

Comment:
Same nit here

Not quite sure if this should be Infof or Debugf... kind of feels like the latter, unless we also want to start logging registration entry changes at Info level

Reply:
Agreed! I believe this should be logged as "debug" :+1: